frontend/src/utils/Tabs.svelte

```diff
@@ -0,0 +1,65 @@
+<style>
+  button {
+    margin-bottom: 0;
+    border: 1px solid white;
```

Comment:
```suggestion
    border: 1px solid var(--main-fg-color);
```

I only noticed this here, but worth grepping for `white` and `black` throughout this PR just to be sure.